modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-tune-help.adoc

```diff
@@ -0,0 +1,27 @@
+= rpk redpanda tune help
+
+Display detailed information about the tuner.
+
+== Usage
+
+[,bash]
+----
+rpk redpanda tune help [TUNER] [flags]
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|-h, --help |- |Help for help.
```

Comment:
^ not sure if "Help for help" is deliberate :)

Reply:
is actually is hahha because the command is called `help`. It´s a self reference. I'll change to rpk redpanda tune help to make it more obvious